api/src/main/java/ca/bc/gov/educ/api/sld/mappers/v1/SldStudentHistoryMapper.java

```diff
@@ -0,0 +1,40 @@
+package ca.bc.gov.educ.api.sld.mappers.v1;
+
+import ca.bc.gov.educ.api.sld.model.SldStudentHistoryEntity;
+import ca.bc.gov.educ.api.sld.struct.v1.SldStudentHistory;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * The interface sld student history mapper.
+ */
+@Mapper(uses = StringMapper.class)
+@SuppressWarnings("squid:S1214")
+public interface SldStudentHistoryMapper {
+  /**
+   * The constant mapper.
+   */
+  SldStudentHistoryMapper mapper = Mappers.getMapper(SldStudentHistoryMapper.class);
+
+
+  /**
+   * To structure sld student history.
+   *
+   * @param sldStudentHistoryEntity the sld student history entity
+   * @return the sld
+   */
+  @Mapping(target = "localStudentId", source="sldStudentHistoryEntity.sldStudentHistoryIdEntity.localStudentId")
```

Comment:
could you please check if mapstruct is generating mappings with null check for these nested mappings